msg/RobotRelative.msg

```diff
@@ -2,8 +2,8 @@
 
 # robot without known color
 uint8 ROBOT_UNDEFINED = 1
-uint8 ROBOT_MAGENTA = 2
-uint8 ROBOT_CYAN = 3
+uint8 ROBOT_RED = 2
```

Comment:
Are these labels utilized elsewhere?

Reply:
Yes, changed them too

Reply:
Todo check the comment from @phinik in this context